### DISCUSSION TITLE
[灌水]赏金好多啊啊啊

### DISCUSSION THREAD

**Discussion by 75020-maker** [share121/inter-knot#697] [其他]
![7effc408b642a00f6e98c127831435e](https://github.com/user-attachments/assets/2bb54f77-e552-4830-9fbd-8a3a39f359c2)
珀尔曼也太值钱了，看到悬赏新闻的我双手微微颤抖
现在问题来了，应该怎么抓他?

**Comment by Hongxuli2006** [comment]
绳匠不知道哦
